src/api/metrics.coffee

```diff
@@ -11,70 +11,66 @@ metrics = require "../metrics"
 # getGlobalLoadTimeMetrics() function for generating aggregated global Metrics #
 ########################################################################
 
-exports.getGlobalLoadTimeMetrics = `function *() {
-
-  var filtersObject = this.request.query;
-  var userRequesting = this.authenticated;
-
-  results = yield metrics.fetchGlobalLoadTimeMetrics(userRequesting, filtersObject);
+exports.getGlobalLoadTimeMetrics =  ->
+  filtersObject = this.request.query
+  userRequesting = this.authenticated
+  results = yield metrics.fetchGlobalLoadTimeMetrics(userRequesting, filtersObject)
   this.body = []
-
-  for (var i = 0; i < results.length; i++) {
-        this.body.push({
-          load: results[i].load,
-          avgResp: results[i].avgResp,
-          timestamp : moment.utc([results[i]._id.year, results[i]._id.month - 1, results[i]._id.day,results[i]._id.hour]).format()//, 'YYYY-MM-DD H+00:00'
-      });
-    }
-
-}`
+  i = 0
+  while i < results.length
```

Comment:
For this you can do the following in coffeescript: `for result, i in results` to get access to `i` (coffeescript is cool :) )